var/spack/repos/builtin/packages/py-h5py/package.py

```diff
@@ -60,8 +61,9 @@ class PyH5py(PythonPackage):
 
```

Comment:
build deps need to be updated:
```toml
[build-system]
requires = [
    "Cython ~=0.29",
    "oldest-supported-numpy",
    "pkgconfig",
    "setuptools >=61",
]
```

Reply:
Looks like setuptools was the only one that needed the update. Thanks!